src/core/pointFeature.js

```diff
@@ -242,14 +242,14 @@ geo.pointFeature = function (arg) {
     var defaultStyle = $.extend(
       {},
       {
-        radius: 10.0,
+        radius: 5.0,
         stroke: true,
-        strokeColor: { r: 0.0, g: 1.0, b: 0.0 },
+        strokeColor: { r: 0.851, g: 0.604, b: 0.0 },
         strokeWidth: 2.0,
```

Comment:
I pixel might be too small but I will see how it looks and if it looks reasonable then I am okay with it.